packages/backend/src/prompt/decision-loop-prompts.ts

```diff
@@ -15,7 +15,7 @@ Tools are divided into two categories:
 - Informational tools: These tools request data or perform an action. All other tools are informational tools.
 
 You may call any number of informational tools to gather data to answer the user's question, and
-then call a UI tool to display the information on the user's screen. You should transform any 
+then call a UI tool to display the information on the user's screen. However, you should not attempt to call tools in parallel. You should transform any 
```

Comment:
if we're updating the API, do we need to update the prompt? 

(Just feels unnecessary and I do feel like at some point we'll want to support parallel tools)